hack/deploy-sbo-local.sh

```diff
@@ -8,32 +8,30 @@ mkdir -p "$OUTPUT"
 
 SBO_LOCAL_LOG="$OUTPUT/sbo-local.log"
 
-_killall(){
-    which killall &> /dev/null
-    if [ $? -eq 0 ]; then
-        killall $1
-    else
-        for i in "$(ps -l | grep $1)"; do if [ -n "$i" ]; then kill $(echo "$i" | sed -e 's,\s\+,#,g' | cut -d "#" -f4); fi; done
-    fi
-}
-
-_killall operator-sdk
-_killall service-binding-operator-local
-
-operator-sdk --verbose run --local --namespace="$OPERATOR_NAMESPACE" --operator-flags "$ZAP_FLAGS" > $SBO_LOCAL_LOG 2>&1 &
+RUN_IN_BACKGROUND="${RUN_IN_BACKGROUND:-false}"
 
-SBO_PID=$!
-
-attempts=24
-while [ -z "$(grep 'Starting workers' $SBO_LOCAL_LOG)" ]; do
-    if [[ $attempts -ge 0 ]]; then
-        sleep 5
-        attempts=$((attempts-1))
-    else
-        echo "FAILED"
-        kill $SBO_PID
-        exit 1
-    fi
-done
+_run_operator(){
+    SERVICE_BINDING_OPERATOR_DISABLE_ELECTION=false WATCH_NAMESPACE="" ./out/operator $ZAP_FLAGS
+}
 
-echo $SBO_PID
+if [ "$RUN_IN_BACKGROUND" == "true" ]; then
+    _run_operator > $SBO_LOCAL_LOG 2>&1 &
+
+    SBO_PID=$!
+
+    attempts=24
+    while [ -z "$(grep 'Starting workers' $SBO_LOCAL_LOG)" ]; do
+        if [[ $attempts -ge 0 ]]; then
+            sleep 5
+            attempts=$((attempts-1))
+        else
+            echo "FAILED"
+            kill $SBO_PID
+            exit 1
+        fi
+    done
```

Comment:
do we really need this, i.e. what we are ensuring with that?

Reply:
We are ensuring that the SBO is actually started and ready for work. That is what is supposed to be meant by `Service Binding Operator is running` step - the part of the step implementation is checking that the reponse is not `FAILED`.
If something goes wrong as part of the "booting", the script will detect that, returns `FAILED` and that will fail the step and fail-fast the whole suite.

Reply:
ok, let's then leave as it is... honestly I am not sure if this improves our confidence that the operator is ready, but that is the topic for another PR. We should come up with a mechanism that would work both in remote and local testing scenarios.